lib/mayaUsd/resources/ae/usdschemabase/custom_image_control.py

```diff
@@ -170,7 +170,7 @@ def filenameBrowser(self, callback):
 
     @staticmethod
     def wantRelativeFileName():
-        opVarName = "mayaUsd_MakePathRelativeToEditTargetLayer"
+        opVarName = "mayaUsd_MakePathRelativeToImageEditTargetLayer"
```

Comment:
New optionVar for the image relative to edit target layer (4th checkbox in new relative section).